dataline-server/src/main/java/io/dataline/server/handlers/ConnectionsHandler.java

```diff
@@ -69,13 +69,18 @@ public ConnectionRead createConnection(ConnectionCreate connectionCreate)
       throws JsonValidationException, IOException, ConfigNotFoundException {
     final UUID connectionId = uuidGenerator.get();
 
+    // todo (cgardens): for MVP we only support full refresh.
+    if (connectionCreate.getSyncMode() != ConnectionCreate.SyncModeEnum.FULL_REFRESH) {
```

Comment:
should we just remove the `APPEND` enum value until then?

Reply:
Not sure. I'm going to merge as-is and we can decide how to handle inputs later.